From 64fe7e543c24259829b290120efc480e9357b04f Mon Sep 17 00:00:00 2001
From: Georges Basile Stavracas Neto <georges.stavracas@gmail.com>
Date: Fri, 26 Feb 2021 19:16:12 -0300
Subject: [PATCH] utils/date-time-utils: Use g_time_zone_new_identifier()

g_time_zone_new() is deprecated in favor of it, so use it.
---
 src/utils/gcal-date-time-utils.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/utils/gcal-date-time-utils.c b/src/utils/gcal-date-time-utils.c
index f7da80b9..06b3d17a 100644
--- a/src/utils/gcal-date-time-utils.c
+++ b/src/utils/gcal-date-time-utils.c
@@ -223,7 +223,7 @@ gcal_date_time_from_icaltime (const ICalTime *date)
 
   zone = i_cal_time_get_timezone (date);
   identifier = zone ? i_cal_timezone_get_location (zone) : NULL;
-  tz = identifier ? g_time_zone_new (identifier) : g_time_zone_new_utc ();
+  tz = identifier ? g_time_zone_new_identifier (identifier) : g_time_zone_new_utc ();
   dt = g_date_time_new (tz,
                         i_cal_time_get_year (date),
                         i_cal_time_get_month (date),
-- 
GitLab

